draft-ietf-mls-protocol.md

```diff
@@ -1000,6 +1000,70 @@ struct {
 } UserInitKey;
 ~~~~~
 
+# Message Framing
+
+[[ TODO prose ]]
+
+~~~~~
+enum {
+    invalid(0),
+    handshake(1),
+    application(2),
+    (255)
+} ContentType;
```

Comment:
Why would a content ever be labelled invalid?

Reply:
The idea is that the ContentType is used to mark where the padding bytes (all zero) end, so it needs to be non-zero.